cpp/create_module/algorithm/create.cpp

```diff
@@ -0,0 +1,56 @@
+#include "create.hpp"
+
+void Create::SetElementProp(mgp::Relationship &element, const mgp::List &prop_key_list,
+                            const mgp::List &prop_value_list, const mgp::RecordFactory &record_factory) {
+  for (size_t i = 0; i < prop_key_list.Size(); i++) {
+    element.SetProperty(std::string(prop_key_list[i].ValueString()), prop_value_list[i]);
+    auto record = record_factory.NewRecord();
+    record.Insert(std::string(kResultRelProp).c_str(), std::move(element));
+  }
+}
+
+void Create::ProcessElement(const mgp::Value &element, const mgp::Graph graph, const mgp::List &prop_key_list,
+                            const mgp::List &prop_value_list, const mgp::RecordFactory &record_factory,
+                            std::unordered_set<mgp::Id> &relIds) {
+  if (element.IsRelationship()) {
+    auto rel = element.ValueRelationship();
+    SetElementProp(rel, prop_key_list, prop_value_list, record_factory);
+    return;
+  }
+  if (element.IsInt()) {
+    relIds.insert(mgp::Id::FromInt(element.ValueInt()));
+    return;
+  }
+  throw mgp::ValueException("First argument must be node, relationship, id or a list of those.");
```

Comment:
Where are we processing nodes here?

Reply:
In last version of these procedures I wanted to use the same ProcessElement for all create functions when merging so I wrote universal exception message but now its not possible anymore but I forgot to change the message haha